lib/hacl-c/AEAD_Poly1305_64.h

```diff
@@ -0,0 +1,101 @@
+/* This file was auto-generated by KreMLin! */
+#include "pprzlink/hacl-c/kremlib.h"
```

Comment:
Here I had to change the original path to match the split of source and header files. Not a big deal if done once, but if there was a cleaner solution (maybe have HACL as a Paparazzi module?) it would be nicer.